python/examples/sum_factor_and_popularity.py

```diff
@@ -0,0 +1,66 @@
+import alpenglow as ag
+import alpenglow.Getter as rs
+import alpenglow.experiments
+import alpenglow.evaluation
+import pandas as pd
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+data = pd.read_csv(
+    "/mnt/idms/recSys/data/30M_lastfm/recoded_online_id_artist_first_filtered",
+    sep=' ',
+    header=None,
+    names=['time', 'user', 'item', 'id', 'score', 'eval'],
+    nrows=100000
+)
+
+class MyExperiment(ag.OnlineExperiment):
+    def _config(self, top_k, seed):
+        model = rs.CombinedModel(**self.parameter_defaults(
+            los_file_name="my_log_file",
+            log_frequency=100000,
+            use_user_weights=False,
+        ))
+        pop_model = rs.PopularityModel()
+        model.add_model(pop_model)
+        pop_updater = rs.PopularityModelUpdater()
+        pop_updater.set_model(pop_model)
+
+        factor_model = rs.FactorModel(**self.parameter_defaults(
+            begin_min=-0.01,
+            begin_max=0.01,
+            dimension=10,
+            initialize_all=False,
+        ))
+
```

Comment:
nem maradt ki egy `model.add_model(factor_model)`?

Reply:
De, csak nem tudom, hogy lehet a pull requestet módosítani.

Reply:
@proto-n A lényeges kérdés az, hogy akarunk-e ilyen példákat a közös kódba. Akarunk? ha igen, akkor mergelek és pótolom a hiányzó sort.

Reply:
szerintem jók igen, csak valszeg kéne majd valamilyen formában szöveget is csatolni hozzájuk, de ezt lehet doksiban is vagy a kód fölé később kommentben